platform/blueprint/builtin/db.go

```diff
@@ -81,6 +81,14 @@ func (db *DB) Save(bp *blueprint.Blueprint) error {
 	if err != nil {
 		return err
 	}
+	check_bp, err := db.BlueprintByName(bp.Name)
+	if err != nil {
+		if !isNotFound(err) {
+			return err
+		}
+	} else if bp.UUID != check_bp.UUID {
+		return errors.New(fmt.Sprintf("Blueprint not saved: same name %s exists", bp.Name))
```

Comment:
Use fmt.Errorf here instead of errors.New + sprintf

Reply:
I just copied the line a couple lines down. I'll change that one, too. :)